docs/source/get_involved/code_of_conduct.rst

```diff
@@ -342,14 +397,161 @@ Guide <https://www.djangoproject.com/conduct/>`__ and we are also
+i.e., valuable possessions, opportunities, qualities, that can be accessed and allocated.
+
+Examples include but are not limited to:
+
+* budget, money, financial credit
+* job opportunities, business opportunities
+* decision making power, soft or hard influence
+* dedicated work time of subordinates or dependents
+* material possessions of value
+* information of value
+* access to any of the above
+
+Best practice
+^^^^^^^^^^^^^
+
+To ensure decision making in line with the code of conduct, decision making must be:
+
+* by the ``sktime`` community, through ``sktime`` community decision making mechanisms.
+  E.g., following decision making outlined in the ``sktime`` governance document.
+* in communication transparent to the ``sktime`` community, following communication requirements outlined in the ``sktime`` governance document.
+* pursuant of and in line with the charitable mission of ``sktime`` and NumFOCUS.
+
+Adherence to widely accepted guidelines of anti-corruption and anti-bribery practice
+(e.g., United Nations Anti-Corruption Guidelines) is strictly expected,
+especially for resource allocation decisions of major magnitude, e.g., in hiring processes,
+or decisions in the 5-digit dollar range and upwards.
+
+This strict requirement for best practice remains unchanged even if a local context may require less,
+e.g., university administrations,
+company policies, national anti-corruption and anti-bribery laws (or their absence), etc.
+
+For instance, it is in-principle possible to misappropriate ``sktime`` resource, while
+being in formal compliance with specific national laws, institutional laws, and local policies.
+Irrespective of this, such an action would still count as a severe CoC violation.
+
+Examples of not acceptable behaviour
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* obtaining a resource opportunity through ``sktime``,
+  then retreating or resigning ``sktime`` roles to declare that the opportunity is now in a personal role and not on behalf of ``sktime``.
+* pretending to communicate on behalf of ``sktime``, or unauthorized use of the ``sktime`` brand,
+  including but not limited to violations of the BSD 3-clause license
+* soft refusal to implement best anti-corruption and anti-bribery practice, e.g.,
+  soft refusal to make budgets, invoives, or spending records transparent to the ``sktime`` community
+* attempts to suppress criticism of bad practice through invoking the code of conduct, e.g., tone policing or DARVO
+
+Investigating CoC violations in resource allocation decisions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+CoC violations when making resource allocation decisions are rarely accompanied by violation of communicative norms.
+
+Such CoC violations can be committed by an individual, or by a networked group of individuals.
+
+DARVO, intimidation, noise generation, and plausible deniability may be common tactics found in an investigation.
+
+It is hence crucial to establish the facts.
```

Comment:
I think it's crucial to establish the facts no matter the violation. I'd reduce this to this one line.
"The investigation's focus must be on the facts."

Reply:
ok, changed, have a look